presidio-anonymizer/presidio_anonymizer/entities/analyzer_result.py

```diff
@@ -17,12 +17,31 @@ class AnalyzerResult:
 
     logger = logging.getLogger("presidio-anonymizer")
 
-    def __init__(self, content: dict):
-        self.__validate_fields(content)
-        self.score = content.get("score")
-        self.entity_type = content.get("entity_type")
-        self.start = content.get("start")
-        self.end = content.get("end")
+    @classmethod
+    def from_json(cls, json: dict):
```

Comment:
Should either be a string (if json) or a dictionary
```suggestion
    def from_json(cls, data: dict):
```